### PR TITLE
Eng 2011 Handle Asynchronosly created conda env in UI

### PR DESCRIPTION
**Pull request by likawind**
## Describe your changes and why you are making these changes
This PR adds basic UI changes to render conda env creation status in UI. In this PR we only adds basics to only render the status. In the next milestone (and before releasing the feature), we will add more details to the page as tracked in https://linear.app/aqueducthq/issue/ENG-2027/[m3]-show-more-details-about-conda-creation

Since `execStatus` is not associated with workflow, we moved previous `WorkflowStatus` file to `execution/chip` for better clarity.

## Related issue number (if any)
ENG-2011

## Loom demo (if any)
https://www.loom.com/share/1dc0308ebeb3416d92cc229f45eaf2c3

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

**Comment by agiron123** [comment]
Small design nit: is there a way we can make the "In Progress" bar width be just as wide as the text "In Progress"?

File: src/ui/common/src/components/execution/chip.tsx
Comment (agiron123) [inline src/ui/common/src/components/execution/chip.tsx:4]: Is this a new component? I'm actually working on a similar component right now in my work for the Data and Workflow table views.

I don't want to hold your code up, but this is something we should consolidate in the future.

**Review by agiron123 — APPROVED** [review]
Code looks good. Left a small design comment above, but this can be merged after resolving that comment.

**Comment by likawind** [comment]
Thanks @agiron123 ! The changes are not finalized, this PR is mainly to set the foundation to render `execState` for conda integration. In follow up PRs, we will render more details and adjust alignments before we merge everything.

So I think you could proceed with your changes and keep me in the loop, I can try to incorporate with them

**Comment by agiron123** [comment]
> Thanks @agiron123 ! The changes are not finalized, this PR is mainly to set the foundation to render `execState` for conda integration. In follow up PRs, we will render more details and adjust alignments before we merge everything.
> 
> So I think you could proceed with your changes and keep me in the loop, I can try to incorporate with them

@likawind Sounds good, I'll let you know when my changes are ready. Should be some time tomorrow.

**Pull request merged**
